watertap/unit_models/tests/test_nanofiltration_ZO.py

```diff
@@ -140,7 +143,7 @@ def test_build(self, unit_frame):
     @pytest.mark.unit
     def test_dof(self, unit_frame):
         m = unit_frame
-        assert degrees_of_freedom(m) == 0
+        check_dof(m, fail_flag=True)
```

Comment:
There's the function `assert_no_degrees_of_freedom` in `watertap.util.initialization` whose implementation is this.

Reply:
OK, replacing now.